src/components/base.scss

```diff
@@ -200,6 +200,11 @@
             text-decoration: underline;
         }
     }
+
+    ul {
```

Comment:
Double check nothing other breaks with this change, since we might be importing a ul in other places.